### PR TITLE
Add landing page for Desktop WPF

### LINKED ISSUE #16009: Update WPF Desktop Guide landing page
The WPF Desktop Guide landing page needs to be updated to the new format.

### PR DESCRIPTION
**Pull request by Thraka**
## Summary

Fixes #16009

### CODE REVIEW

File: docs/desktop-wpf/index.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+### YamlMime:Landing
+
+title: Windows Presentation Foundation (WPF) documentation
+summary: Learn about Windows Presentation Foundation, an open-source, graphical user interface for Windows.
```
Comment (gewarren) [inline docs/desktop-wpf/index.yml:4]:
```suggestion
summary: Learn about using Windows Presentation Foundation, an open-source, graphical user interface for Windows, on .NET Core.
```

File: docs/desktop-wpf/index.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+metadata:
+  title: Windows Presentation Foundation (WPF) for .NET Core documentation
+  description: Learn about Windows Presentation Foundation, an open-source, graphical user interface for Windows.
```
Comment (gewarren) [inline docs/desktop-wpf/index.yml:8]:
```suggestion
  description: Learn about using Windows Presentation Foundation, an open-source, graphical user interface for Windows, on .NET Core.
```

File: docs/desktop-wpf/index.yml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        links:
+          - text: Windows Presentation Foundation overview
+            url: overview/index.md
+          - text: XAML in Windows Presentation Foundation
+            url: fundamentals/xaml.md
+
+  # Card
+  - title: Get started with WPF
+    linkLists:
+      - linkListType: overview
```
Comment (gewarren) [inline docs/desktop-wpf/index.yml:29]: These articles don't sound like overview to me.

```suggestion
      - linkListType: concept
```

File: docs/desktop-wpf/index.yml [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+            url: migration/convert-project-from-net-framework.md
+      - linkListType: tutorial
+        links:
+          - text: Create a WPF application
+            url: https://docs.microsoft.com/visualstudio/get-started/csharp/tutorial-wpf?toc=/dotnet/desktop-wpf/toc.json&bc=/dotnet/breadcrumb/toc.json
+
+  # Card
+  - title: Fundamentals - XAML
+    linkLists:
+      - linkListType: overview
```
Comment (gewarren) [inline docs/desktop-wpf/index.yml:43]:
```suggestion
      - linkListType: concept
```

File: docs/desktop-wpf/index.yml [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+            url: fundamentals/styles-templates-overview.md
+      - linkListType: how-to-guide
+        links:
+          - text: Create and apply a style
+            url: fundamentals/styles-templates-create-apply-style.md
+          - text: Create and apply a template
+            url: themes/how-to-create-apply-template.md
+      - linkListType: reference
+        links:
+          - text: XAML Language Reference
```
Comment (gewarren) [inline docs/desktop-wpf/index.yml:57]:
```suggestion
          - text: XAML language reference
```

File: .openpublishing.redirection.json [hunk 6]
```diff
@@ -1648,10 +1648,6 @@
       "source_path": "docs/desktop-wpf/getting-started/index.md",
       "redirect_url": "https://docs.microsoft.com/visualstudio/get-started/csharp/tutorial-wpf?toc=/dotnet/desktop-wpf/toc.json&bc=/dotnet/breadcrumb/toc.json"
     },
-    {
-      "source_path": "docs/desktop-wpf/index.md",
```
Comment (gewarren) [inline .openpublishing.redirection.json:1652]: I don't get this. There is no index.md file in the desktop-wpf folder. I think you can remove this entry.

Comment (gewarren) [reply .openpublishing.redirection.json:1652]: Oh I see, you're actually removing it. NVM.

File: docs/desktop-wpf/index.yml (hunk 4 above)
Comment (gewarren) [reply docs/desktop-wpf/index.yml:43]: Actually only "Define XAML resources" should go under a concept list type. Apologies.

Comment (Thraka) [reply docs/desktop-wpf/index.yml:43]: Thanks, I forgot to check these.

File: docs/desktop-wpf/index.yml [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+            url: overview/index.md
+          - text: XAML in Windows Presentation Foundation
+            url: fundamentals/xaml.md
+
+  # Card
+  - title: Get started with WPF
+    linkLists:
+      - linkListType: concept
+        links:
+          - text: Differences from .NET Framework
```
Comment (gewarren) [inline docs/desktop-wpf/index.yml:31]:
```suggestion
          - text: Differences from WPF on .NET Framework
```

Comment (gewarren) [reply docs/desktop-wpf/index.yml:31]: Otherwise it seems like it's comparing WPF to .NET Framework.

**Review by gewarren — APPROVED** [review]
Sorry one last suggestion that I meant to make before.

**Pull request merged**